pkg/setting/setting_session_test.go

```diff
@@ -25,7 +25,7 @@ func TestSessionSettings(t *testing.T) {
 		})
 		require.Nil(t, err)
 
-		require.Equal(t, 1, logger.WarnLogs.Calls)
```

Comment:
Why / what has changed here? Want to double check this is intentional.

Reply:
we now always return a warning from setting up the configuration. This warning is part of setting the `OAuthSkipOrgRuleSync` to true